include/rpi/hardware-control-implementation.h

```diff
@@ -50,15 +51,18 @@ class GPIO {
   // not possible.
   double StartClock(double frequency_hertz);
   void StopClock();
-
+  
   // Switches the output of the currently running clock.
   void EnableClockOutput(bool b);
+  void SetTxPower(CarrierPower power);
 
  private:
-  volatile uint32_t *gpio_port_;
-  volatile uint32_t *gpio_set_bits_;
-  volatile uint32_t *gpio_clr_bits_;
-  volatile uint32_t *clock_reg_;
+  volatile uint32_t *gpio_port_ = nullptr;
+  volatile uint32_t *gpio_set_bits_= nullptr;
+  volatile uint32_t *gpio_clr_bits_= nullptr;
+  volatile uint32_t *clock_reg_= nullptr;
 };
 
-#endif  // RPI_GPIO_H
+using GPIO = HardwareControl::Implementation;
+
+#endif
```

Comment:
Please make sure to have a newline at the end of each file (the .editorconfig file also will configure that)